From 88f97dbf80b1052850033a4c0847dbd41d2d226a Mon Sep 17 00:00:00 2001
From: George Zacharia <george.zcharia@gmail.com>
Date: Sat, 4 Apr 2020 12:48:46 +0530
Subject: [PATCH] Extend max notification icons to statusbar icons

Some devices at high DPI can accommodate much more icons
Change-Id: If6eea3a9321d3b4598f13310525d8cc741941cb0
---
 .../android/systemui/statusbar/phone/StatusIconContainer.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusIconContainer.java b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusIconContainer.java
index 60af06920e24..2b8764553f72 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusIconContainer.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusIconContainer.java
@@ -51,8 +51,8 @@ public class StatusIconContainer extends AlphaOptimizedLinearLayout {
     private static final String TAG = "StatusIconContainer";
     private static final boolean DEBUG = false;
     private static final boolean DEBUG_OVERFLOW = false;
-    // Max 8 status icons including battery
-    private static final int MAX_ICONS = 7;
+    // Max 8 status icons including battery, but some devices can accomodate even more
+    private final int MAX_ICONS = getResources().getInteger(R.integer.config_maxVisibleNotificationIcons);
     private static final int MAX_DOTS = 1;
 
     private int mDotPadding;
